test/unit/token/Tranche.t.sol

```diff
@@ -175,47 +175,24 @@ contract TrancheTokenTest is Test {
         assertEq(token.balanceOf(self), amount);
     }
 
-    // mint
+    // // mint
     function testMintTokensToMemberWorks(uint256 amount) public {
-        amount = bound(amount, 0, type(uint256).max / 2);
+        amount = bound(amount, 0, type(uint128).max / 2);
 
         // mint fails -> self not a member
-        vm.expectRevert(bytes("RestrictionManager/destination-not-a-member"));
+        vm.expectRevert(bytes("destination-not-a-member"));
         token.mint(targetUser, amount);
 
-        restrictionManager.updateMember(targetUser, uint64(validUntil));
-        (, uint64 actualValidUntil) = restrictionManager.restrictions(targetUser);
-        assertEq(actualValidUntil, validUntil);
+        restrictionManager.updateMember(address(token), targetUser, uint64(validUntil));
+        assertTrue(restrictionManager.isMember(address(token), targetUser));
 
         token.mint(targetUser, amount);
         assertEq(token.balanceOf(targetUser), amount);
         afterTransferAssumptions(address(0), targetUser, amount);
 
         vm.warp(validUntil + 1);
 
-        vm.expectRevert(bytes("RestrictionManager/destination-not-a-member"));
-        token.mint(targetUser, amount);
-    }
-
-    function testTransferMintFailsNoPermissionOnRestrictionManager() public {
```

Comment:
Does not apply anymore since the tranche token is not anymore a ward on the hook/restriction manager